laia/decoders/ctc_greedy_decoder.py

```diff
@@ -6,12 +6,28 @@
 
 
 class CTCGreedyDecoder:
+    """
+    Initialize a gready CTC decoder.
+    Args:
+        segmentation (bool): whether to output CTC segmentation info.
```

Comment:
This argument is not used in `__init__` ?

Reply:
I initially tried to move `segmentation` and `apply_softmax` to the constructor, but that broke the callbacks, so I gave up.
I removed the docstring.